pkg/awsutils/vpc_ip_resource_limit.go

```diff
@@ -278,26 +278,26 @@ var InstanceIPsAvailable = map[string]int64{
 	"m5.2xlarge":    15,
 	"m5.4xlarge":    30,
 	"m5.12xlarge":   30,
-	"m5.24xlarge":   50,
+	"m5.24xlarge":   31,
 	"m5a.large":     10,
 	"m5a.xlarge":    15,
 	"m5a.2xlarge":   15,
 	"m5a.4xlarge":   30,
 	"m5a.12xlarge":  30,
-	"m5a.24xlarge":  50,
+	"m5a.24xlarge":  31,
 	"m5d.large":     10,
 	"m5d.xlarge":    15,
 	"m5d.2xlarge":   15,
 	"m5d.4xlarge":   30,
 	"m5d.12xlarge":  30,
-	"m5d.24xlarge":  50,
+	"m5d.24xlarge":  31,
 	"p2.xlarge":     15,
 	"p2.8xlarge":    30,
 	"p2.16xlarge":   30,
 	"p3.2xlarge":    15,
 	"p3.8xlarge":    30,
 	"p3.16xlarge":   30,
-	"p3dn.24xlarge": 50,
+	"p3dn.24xlarge": 31,
```

Comment:
Lets only reduce it for `p3dn.24xlarge`

Reply:
I split this into two commits.  One for the p3dn.24xlarge and one for all others.  This way we can cherry-pick the p3dn fix into 1.3.1 (since it is not currently in 1.3.0 it won't break anything).